boards/stm32f746g-disco/include/periph_conf.h

```diff
@@ -94,7 +94,7 @@ static const uart_conf_t uart_config[] = {
         .tx_af      = GPIO_AF7,
         .bus        = APB2,
         .irqn       = USART1_IRQn,
-#ifdef MODULE_PERIPH_DMA
+#if IS_USED(MODULE_PERIPH_DMA)
```

Comment:
`IS_USED()` is only needed in C code, when we can use `#ifdef` we should just do so for clarity.